build.cake

```diff
@@ -347,10 +368,9 @@ Task("PackageZip")
             GetFiles(CurrentImageDir + "bin/net35/**/*.*") +
             GetFiles(CurrentImageDir + "bin/net40/**/*.*") +
             GetFiles(CurrentImageDir + "bin/net45/**/*.*") +
-            GetFiles(CurrentImageDir + "bin/netstandard1.4/**/*.*") +
             GetFiles(CurrentImageDir + "bin/netstandard2.0/**/*.*") +
-            GetFiles(CurrentImageDir + "bin/netcoreapp1.1/**/*.*") +
-            GetFiles(CurrentImageDir + "bin/netcoreapp2.0/**/*.*");
```

Comment:
Are this changes also correcting some older changes, where we overlooked this section of the code?